### ISSUE TITLE
E-mail "to whom" signature - needed when you have more addresses receiving messages

### ISSUE THREAD

**Issue by lukaszpolowczyk** [inbasic/ignotifier#643]
I have more email addresses where I receive messages.  This is supported by this feature - https://support.google.com/mail/answer/138350?hl=pl&authuser=1

 In the ignotifier extension popup, I don't see what email address I received the message to.
 I have to go to GMail to see this.

 So please add information in the extension popup to which email address the message was sent.

**Comment by inbasic** [comment]
Do you get any info needed in the "feed"? https://mail.google.com/mail/u/0/feed/atom/inbox

**Comment by lukaszpolowczyk** [comment]
@inbasic No.  This must be information assigned to a specific message. I think so.
